load-jetpack.php

```diff
@@ -44,7 +44,7 @@ function jetpack_should_use_minified_assets() {
 require_once JETPACK__PLUGIN_DIR . 'class.jetpack-client-server.php';
 require_once JETPACK__PLUGIN_DIR . 'class.jetpack-user-agent.php';
 require_once JETPACK__PLUGIN_DIR . 'class.jetpack-post-images.php';
-require_once JETPACK__PLUGIN_DIR . 'class.jetpack-error.php';
+require_once JETPACK__PLUGIN_DIR . 'class-jetpack-error.php';
```

Comment:
I think it would have been ok to delete `Jetpack_Error` if you had also deleted this line.

Reply:
True. 90cbb0ad8